desc/coils.py

```diff
@@ -188,6 +253,40 @@ class FourierPlanarCoil(Coil, FourierPlanarCurve):
     name : str
         name for this coil
 
+    Examples
+    --------
+    .. code-block:: python
+
+        from desc.coils import FourierPlanarCoil
+        from desc.grid import LinearGrid
+        import numpy as np
+
+        I = 10
+        mu0 = 4 * np.pi * 1e-7
+        R_coil = 10
+        # circular coil given by center at (0,0,0)
+        # and normal vector in Z direction (0,0,1) and radius 10
+        coil = FourierPlanarCoil(
+            current=I,
+            center=[0, 0, 0],
+            normal=[0, 0, 1],
+            r_n=R_coil,
+            modes=[0],
+            grid=LinearGrid(N=100),
```

Comment:
FYI, I'm not sure about having `grid` objects assigned to each coil. I was considering getting rid of it in #583 but felt we should discuss it first. I think it would be better to add something like `source_grid` as an optional argument to `compute_magnetic_field`, especially since after #583 whatever grid you assign to the coil won't be used by the underlying `Curve.compute` method.

Reply:
I agree, that is something we can change in #494 when I update the coil stuff to reflect the #583 changes. This PR though is just for the docs